harper-core/tests/text/linters/Alice's Adventures in Wonderland.snap.yml

```diff
@@ -1491,6 +1630,17 @@ Message: |
 
 
 
+Lint:    Miscellaneous (0 priority)
+Message: |
+    1226 | up into a sort of knot, and then keep tight hold of its right ear and left foot,
+         |                                                                            ^~~~ Consider adding an article before this noun.
```

Comment:
Here "foot" and even "left foot" are part of a list making up a noun phrase that as a group has the determiner "its". This gets deeper and deeper into syntax parsing!

For now you can see the adjective "left" makes it not require an article.